cognite/client/data_classes/data_modeling/cdm/v1.py

```diff
@@ -406,14 +406,14 @@ def __init__(
         existing_version: int | None = None,
         type: DirectRelationReference | tuple[str, str] | None = None,
     ) -> None:
-        TypedNodeApply.__init__(self, space, external_id, existing_version, None, type)
+        TypedNodeApply.__init__(self, space, external_id, existing_version, type)
         self.name = name
         self.description = description
         self.tags = tags
         self.aliases = aliases
 
 
-class CogniteDescribableNode(_CogniteDescribableProperties, TypedNode):
+class CogniteDescribable(_CogniteDescribableProperties, TypedNode):
```

Comment:
Is the thinking that the node version of Describable is without suffix node, while we keep it for edge?

Reply:
Ohh. I didn't realize we had a separate one for Edge. My bad, then it makes sense to keep it for both. I'll revert this commit.